pkg/utils/path/path.go

```diff
@@ -39,3 +40,15 @@ func ExpandHome(path string) string {
 
 	return Join(home, path[1:])
 }
+
+// Expand expands absolute directory in file paths.
+func Expand(path string) (string, error) {
```

Comment:
We can merge the function of ExpandHome into Expand, both of them are actually the same function which is to expand the path.